lib/exceptional_synchrony/event_machine_proxy.rb

```diff
@@ -42,7 +42,9 @@ def sleep(seconds)
     end
 
     def yield_to_reactor
-      @synchrony.sleep(0)
+      if reactor_running?
+        @synchrony.sleep(0)
```

Comment:
I presume .sleep(0) throws an exception when synchrony is not running. (Generally this should be safe...)